<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 243, 255);
width: 100%;
height: 100%;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="mainWindowWidget" native="true">
      <layout class="QVBoxLayout" name="mainWindow">
       <item>
        <widget class="QWidget" name="top" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(77, 169, 255);
height: 50%;
margin: 0px 0px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="topDescriptionHorizontalLayout">
            <item>
             <widget class="QLabel" name="headerLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 24px;
color: darkblue;
font-weight: 450;</string>
              </property>
              <property name="text">
               <string>Saber-Chess</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="topPlayersButton">
              <property name="maximumSize">
               <size>
                <width>397</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>TOP Players</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="communityButton">
              <property name="styleSheet">
               <string notr="true">margin-right: 90%;</string>
              </property>
              <property name="text">
               <string>Community</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="settingsButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">margin-left: 90%;</string>
              </property>
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="centerWidget" native="true">
         <layout class="QHBoxLayout" name="center">
          <item>
           <layout class="QHBoxLayout" name="main">
            <item>
             <layout class="QVBoxLayout" name="ChatsVerticalLayout">
              <item>
               <widget class="QLabel" name="chatsListLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(30, 31, 95);
padding: 0px 100%;</string>
                </property>
                <property name="text">
                 <string>Chats</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="searchChathorizontalLayout">
                <item>
                 <widget class="QLabel" name="searchChatLabel">
                  <property name="text">
                   <string>Search:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="searchChatLine">
                  <property name="styleSheet">
                   <string notr="true">height: 20%;
margin: 0px 20px;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="searchChatButton">
                  <property name="styleSheet">
                   <string notr="true">height: 40%;
margin: 5%;
background-color: rgb(138, 226, 52);</string>
                  </property>
                  <property name="text">
                   <string>search</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="ChatListWidget"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="MainVerticalLayout">
              <item>
               <widget class="QPushButton" name="playButton">
                <property name="styleSheet">
                 <string notr="true">margin: 0 40%;
margin-top: 130%;
height: 50%;
background-color:rgb(115, 210, 22);
border: 2px solid rgb(27, 156, 0);</string>
                </property>
                <property name="text">
                 <string>Play</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkRandomPlayer">
                <property name="styleSheet">
                 <string notr="true">height: 15%;</string>
                </property>
                <property name="text">
                 <string>Check random player on rating</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelPreviewPhoto">
                <property name="styleSheet">
                 <string notr="true">padding: 0 100%;</string>
                </property>
                <property name="text">
                 <string>Preview photo</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="friendsVerticalLayout">
              <item>
               <widget class="QLabel" name="friendsListLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(30, 31, 95);
padding: 0px 100%;</string>
                </property>
                <property name="text">
                 <string>Friends list</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="searchChatHorizontalLayout">
                <item>
                 <widget class="QLabel" name="searchFriendLabel">
                  <property name="text">
                   <string>Search:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="searchFriendLIne">
                  <property name="styleSheet">
                   <string notr="true">height: 20%;
margin: 0px 20px;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="searchFriendButton">
                  <property name="styleSheet">
                   <string notr="true">height: 40%;
margin: 5%;
background-color: rgb(138, 226, 52);</string>
                  </property>
                  <property name="text">
                   <string>search</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListView" name="listView"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="gameWindowWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>LeaveGame</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottom" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(77, 169, 255);
height: 50%;
margin: 0px 0px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="aboutGameButton">
            <property name="text">
             <string>ABOUT GAME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="developersButton">
            <property name="text">
             <string>developers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="donateButton">
            <property name="text">
             <string>donate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="contactsButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>contacts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="QuitButton">
            <property name="maximumSize">
             <size>
              <width>397</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
